changelog/unreleased/enhancement-allow-listing-regular-users.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: Allow regular users to list other users
+
+Regular users can search for other users, but having some limitations
```

Comment:
```suggestion
Regular users can search for other users, with some limitations.
```

Reply:
Maybe list the limitations in the changelog item as it has high impact

Reply:
done